### ISSUE TITLE
Better error message

### ISSUE THREAD

**Issue by jaumard** [trailsjs/smokesignals#6]
When I develop a Trailpack, I don't want to `npm link` it directly to a project to test it (I prefer do it when the trailpack is almost finish). So I only use mocha test to run/debug my Trailpack. 
When an error occurred I have this kind of log : 

```
smokesignal logger: [ error ] [TypeError: Cannot read property 'getOptionsFromQuery' of undefined]
```

Is it possible to add the file and line number where the error occurred ? Could be useful :) 
Thanks

**Comment by scott-wyatt** [comment]
This would be extremely useful for me as well.  Can we get #11 merged?

**Comment by tjwebb** [comment]
resolved in #11.

thanks @jaumard!
